extensions/calendly/package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "calendly",
+  "title": "Calendly",
+  "description": "Raycast commands to access Calendly instance",
+  "icon": "logo.png",
+  "author": "eluce2",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "calendly",
+      "title": "Calendly",
+      "subtitle": "Copy Meeting URL",
+      "description": "Copy Meeting URL",
+      "mode": "view"
+    }
+  ],
+  "dependencies": {
+    "@raycast/api": "^1.25.0",
+    "axios": "^0.23.0",
+    "lodash": "^4.17.21"
+  },
+  "preferences": [
+    {
+      "name": "token",
+      "type": "password",
+      "required": true,
+      "title": "Calendly Personal Access Token",
```

Given that this is already in the Calendly context, I would remove the suffix. This should also make it fit on one line.
```suggestion
      "title": "Personal Access Token",
```